examples/vision-camera/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "vision-camera",
+  "version": "1.0.0",
+  "main": "index.ts",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@fishjam-cloud/react-native-client": "*",
+    "expo": "~52.0.42",
+    "expo-status-bar": "~2.0.1",
+    "react": "18.3.1",
+    "react-native": "0.76.8",
+    "react-native-vision-camera": "^4.6.4",
+    "react-native-worklets-core": "^1.5.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.26.10",
+    "@types/react": "~18.3.12",
+    "typescript": "^5.3.3"
```

Comment:
why so old?

Reply:
It's the default when you create a new expo app, but good catch. I'll make it the same as other repos.